cockatrice/src/dlg_settings.cpp

```diff
@@ -1208,8 +1217,8 @@ DlgSettings::DlgSettings(QWidget *parent) : QDialog(parent)
 #else
     QRect rec = QApplication::desktop()->availableGeometry();
 #endif
-    this->setMinimumSize(rec.width() / 2, rec.height() - 100);
-    this->setBaseSize(rec.width(), rec.height());
+    this->setMaximumSize(rec.width() - 100, rec.height() - 100);
+    this->setMinimumSize(700, 600);
```

Comment:
I don't get why we added a min size, what if they have a super small screen?

Reply:
>what if they have a super small screen?

That would be very difficult (impossible?) to accomplish: https://en.wikipedia.org/wiki/List_of_common_resolutions

The idea of this min size is to **guarantee readability of all UI elements**, labels and boxes in the various settings tabs while trying to ensure functionality on basically all display resolutions. It's not a random value.

The former /2 logic looked very random and did not guarantee a uniform layout. It even forces you to use an oversized settings dialog on modern screens without having a benefit.
I'm open for other numbers or a different logic of course